part-spec/ic_io.json

```diff
@@ -46,6 +46,78 @@
                 }
             }
         },
+        {
+            "title": "retimer",
+            "type": "object",
+            "required": [
+             "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "numberOfLanes": {
+                    "description": "number of lanes supported by a device",
+                    "type": "number"
+                },
+                "interface": {
+                    "description": "list of interface(s) supported by a device",
```

Comment:
If this is a "list", shouldn't the type be an array of string, rather than just a string?